cypress/drafts/Transfer/drag-and-drop-items.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Drag and drop selected items
+
+    Background:
+        Given reordering is enabled
+
+    # I have excluded any design information from this scenario: opacity, how the items look when dragging etc.
+    Scenario: The user clicks, drags and drops a single item
+        Given the selected list has two or more items
+        When the user clicks and drags a single item
+        Then the item should be removed from its original position in the list
+        And the item should be placed in the list at the cursor position when the click is released
+
+    Scenario: The user clicks, drags and drops multiple items
+        Given the selected list has three or more items
+        And the selected list has two or more highlighted items
+        When the user clicks and drags one of the highlighted items to a new position
+        Then the items should be removed from their original positions
+        And the items should be placed in the list at the cursor position when the click is released
+        And the items should be ordered based on their previous ordering
+
+    Scenario: The user clicks, drags and drops a single item in a list of one
+        Given the selected list has one item
+        And the selected list has one highlighted item
+        When the user clicks, drags and drops the highlighted item
+        Then the highlighted item should remain in position and shouldn't not move
+
+
+    # an attempt an outline of how multiple items reorder on drag and drop
```

Comment:
I'd change the wording slightly, `an attempt to outline how multiple items reorder on drag and drop`